packages/components/src/tab-panel/test/index.tsx

```diff
@@ -165,18 +166,18 @@ describe.each( [
 				</SlotFillProvider>
 			);
 
-			expect( getSelectedTab() ).not.toHaveTextContent( 'Alpha' );
+			expect( await getSelectedTab() ).not.toHaveTextContent( 'Alpha' );
 			expect( mockOnSelect ).toHaveBeenCalledTimes( 1 );
 			expect( mockOnSelect ).toHaveBeenLastCalledWith( 'alpha' );
-			await expect( getSelectedTab() ).not.toHaveFocus();
+			await expect( await getSelectedTab() ).not.toHaveFocus();
```

Comment:
I'm seeing a lot of warnings in my IDE that the first `await` on lines like these are unnecessary.

```suggestion
		expect( await getSelectedTab() ).not.toHaveFocus();
```

Reply:
Good catch, I missed those! Thanks! Removed all unneeded `awaits` in 5cb80aed87 and tests are still passing locally 👍